modules/examiner/php/addexaminer.class.inc

```diff
@@ -64,6 +64,21 @@ class AddExaminer extends \NDB_Page
         $fullName = $values['addName'] ?? '';
         $siteID   = $values['addSite'] ?? '';
 
+        // check for site permissions
+        $user = \User::singleton();
+        if (!$user->hasAnyPermission(
+            [
+                'superuser',
+                'examiner_multisite',
+            ]
+        )
+            && !in_array($siteID, $user->getCenterIDs())
+        ) {
+            return  new \LORIS\Http\Response\JSON\Conflict(
```

Comment:
This situation is a `403 Forbidden` not a `409 Conflict` situation. There is no conflict with editing the state of the resource, access is forbidden to it.